src/System.Diagnostics.Process/tests/ProcessTest_ConsoleApp/ProcessTest_ConsoleApp.cs

```diff
@@ -7,6 +8,7 @@ class Program
     {
         static int Main(string[] args)
         {
+            Thread.CurrentThread.CurrentUICulture = new CultureInfo("en-US");
             try
```

Comment:
@stephentoub  @weshaggard  I believe that changing locale makes test do what it is intended to do - test if app launches and prints message, and hosts receives *exactly* that message. This functionality is not locale-dependent itself and thus it must be tested with locale locked to get 100% predictable result.
If we check using string.Contains() or something like that this test will be pointless - it may contain some garbage at beginning or at the end of the string and still pass the test.

As an option - since comment clearly says that "We're testing STDERR streams" it is possible not to throw-catch-print exception but to write some pre-defined message that will allow to check if Error stream is working fine. This will allow to drop dependency on Exception.ToString() implementation details.